main.go

```diff
@@ -35,16 +35,21 @@ func handleCommand(command string) {
 	case "cart":
 		fmt.Println("Your cart is currently empty.")
 	case "quit":
-		break
+		fmt.Println("(exit the program with code 0)")
 	default:
 		fmt.Println("Unrecognized command. Command should be one of: menu, cart, quit.")
 	}
 }
 func main() {
 	var msg string
-	fmt.Printf("Welcome to %s\n", RESTAURANT_NAME)
-	for {
+	option := true
+	fmt.Printf("Welcome to %s\n> ", RESTAURANT_NAME)
+	for option {
 		fmt.Scanln(&msg)
 		handleCommand(msg)
+		fmt.Print("> ")
+		if msg == "quit" {
```

Comment:
Handling the "quit" message here does not feel like a good design as it spreads the logic between two functions: `main` and `handleCommand`. If you need to exit the program in `main()` rather than using `os.Exit` (for example, because you need to do some cleanup work in a deferred function call), a better choice would be returning a boolean value from `handleCommand`.